helm-chart/sefaria-project/templates/configmap/local-settings-file.yaml

```diff
@@ -179,11 +179,10 @@ data:
 
     # Turns off search autocomplete suggestions, which are reinitialized on every server reload
     # which can be annoying for local development.
-    DISABLE_AUTOCOMPLETER = False
-    ENABLE_LINKER = False
+    DISABLE_AUTOCOMPLETER = bool(os.getenv("DISABLE_AUTOCOMPLETER", False))
```

Comment:
Do we want the default value here to be `False`? Web pods, which are arguably the standard use-case, will set this to `True`.

Reply:
I thought web pods generally have the autocompleter enabled?  So it should not be disabled?